Sprint-1/interpret/to-pounds.js

```diff
@@ -1,21 +1,28 @@
 const penceString = "399p";
+// a price in pence 
 
 const penceStringWithoutTrailingP = penceString.substring(
   0,
   penceString.length - 1
 );
+// remove "p" from the string
 
 const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+// make sure the string has at least 3 digits by adding 0 at the front if needed
 const pounds = paddedPenceNumberString.substring(
   0,
   paddedPenceNumberString.length - 2
 );
+//extracts the pounds part
 
 const pence = paddedPenceNumberString
   .substring(paddedPenceNumberString.length - 2)
   .padEnd(2, "0");
```

Comment:
Would this code work equally well if we remove `.padEnd(2, "0")`? It seems redundant.

Reply:
I deleted it and yes there wasn't any need for it!